src/javascript/JContent/ContentNavigation/NavigationHeader/SwitchersLayout/SiteSwitcher.jsx

```diff
@@ -95,7 +95,7 @@ class SiteSwitcher extends React.Component {
                         return null;
                     }
 
-                    if (loading) {
+                    if (loading || (data && data.jcr === null)) {
```

Comment:
I am a bit confuse by this change. What method is failing exactly? Isn't it the call to getSites(data). If that is so, what about changing line 51 (in getSites() function) instead to check for data.jcr not being null?

Reply:
This was a workaround until we found the reason of the issue,
It was added as the loading behaviour because it should display the same empty component when it happens.

Reply:
The main issue has been fixed